named_entity_recognition_bert/Legal-NER.ipynb

```diff
@@ -0,0 +1,1802 @@
+   "metadata": {},
+   "source": [
+    "This tutorial shows an example of solving ```Named Entity Recognition task``` with [SuperAnnotate](https://www.superannotate.com/) and [HuggingFace](https://huggingface.co/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40de6662-f702-4831-8731-30e2e923d673",
+   "metadata": {},
+   "source": [
+    "The main goal of this tutorial is to show how one could annotate some part of data with ```SuperAnnotate``` tools and then build a model with ```HuggingFace``` to automatically annotate the rest of data and upload new annotations to [SuperAnnotate platform](https://app.superannotate.com/). These automatically generated annotations may be additionaly checked and modified manually."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "31799dad-814b-470b-9eb2-0c46f190fcba",
+   "metadata": {},
+   "source": [
+    "All the experiments described in this tutorial were done with [Legal NER](https://paperswithcode.com/dataset/legal-ner) dataset. It is a corpus of 46545 annotated legal named entities mapped to 14 legal entity types. It is designed for named entity recognition in indian court judgement."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9332327",
+   "metadata": {},
+   "source": [
+    "![](../docs/legal-ner/folders_legal_ner.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51494c70-acf6-42c3-b7fb-26e400670a2b",
+   "metadata": {},
+   "source": [
+    "The tutorial starts with the assumption that we have partially annotated dataset of texts.\n",
+    "The data is stored on S3 bucket and splitted into two parts: \n",
+    "* $\\textbf{train}$ (~40%) $-$ annotated data for training\n",
+    "* $\\textbf{unlabeled}$ (~60%) $-$ data that will be annotated by the model\n",
+    "\n",
+    "These folders are connected with existing project on [SuperAnnotate platform](https://app.superannotate.com/) and train dataset has already been annotated manually. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "17d9eca6",
+   "metadata": {},
+   "source": [
+    "![](../docs/legal-ner/ner_text_example.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5dc6e1c5-ad12-43ca-a0f8-82abec50f3d7",
+   "metadata": {},
+   "source": [
+    "In the examples below we used ```SuperAnnotate SDK```, ```Boto3 SDK``` and ```HuggingFace```. $\\ $\n",
+    "Some parts of code used here are provided as examples in [SuperAnnotate](https://doc.superannotate.com/docs/getting-started), [Boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html) and  [HuggingFace](https://huggingface.co/) documentations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95907ec6-755e-47bb-a7b2-388a19f9e50c",
+   "metadata": {},
+   "source": [
+    "In this tutorial we will go through the following steps:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "81483c83",
+   "metadata": {},
+   "outputs": [],
+   "source": []
```

Comment:
In this tutorial we will go through the following steps:
And empty cell without steps (you should add them)